its/autoscan/src/test/resources/autoscan/diffs/diff_S5838.json

```diff
@@ -1,6 +1,6 @@
 {
   "ruleKey": "S5838",
   "hasTruePositives": false,
-  "falseNegatives": 274,
+  "falseNegatives": 280,
```

Comment:
This looks suspicious. The number of issues should not change, but we only want a specialized report and quick fix for some of them.